### DISCUSSION TITLE
.me domain

### DISCUSSION THREAD

**Discussion by ayushsisodia7** [community/community#58743] [Programming Help]
### Hi everyone,

Recently, I have enrolled in Student Developer Pack.
I have got I year .me domain on NameCheap.
I have create my own domain on it. 
What is its use now and how to use it?

**Comment by j0ashm** [comment]
@ayushsisodia7 :wave: 
Congratulations on getting your domain!
Using it depends on how your website is setup, if you're using GitHub Pages to serve your domain, check out [this](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site) page from the GitHub docs for instructions on how you can setup your custom domain with GitHub Pages.

If you're not using GitHub Pages, you can probably deploy your website using services like [Vercel](https://vercel.com) or [Netlify](https://netlify.com), however this is subjective based on the technologies your website uses so take my answer with a grain of salt.

Feel free to let me know if you have more questions!

**Reply by NamanOP1408** [reply]
Hii, i also got a .me domain and it is hosted on github pages but i want to host it on hostinger, i already have a hosting plan on hostinger. is it possible to do that?

**Reply by j0ashm** [reply]
@NamanOP1408 I believe that is highly dependent on what kind of hosting service Hostinger provides. I'm not sure if you have a cPanel plan, or if it is a Linux VM.
Hostinger should have guides on how to host a website, or just generally googling the type of machine you have should give you some useful results.

**Reply by NamanOP1408** [reply]
![Screenshot 2023-07-07 195941](https://github.com/community/community/assets/138278102/eb86490f-9a0a-4e35-8c44-b6aeefd8b637)
i connected the domain to my hosting and i installed wordpress and when i click editsite it gives me this error, i have hpanel hosting

**Reply by j0ashm** [reply]
You wouldn't have a GitHub Pages site if you linked the domain to Hostinger.

AFAIK, Wordpress isn't even supported with GitHub Pages.

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @ayushsisodia7 , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
